variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,756 @@
+# List of variables related to power plants
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term cost   [€/GWh] = Linear Var Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
+# constant term cost   [€/h]   = Constant Var Cost [Mcal/h] * Price|Primary Energy [€/Mcal]
+Constant Var Cost|Electricity|Coal:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Constant Var Cost|Electricity|Gas:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Constant Var Cost|Electricity|Nuclear:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Constant Var Cost|Electricity|Oil:
+   description: Constant term (intercept) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the unit commitment variable of each power plant per unit time.
+   unit: Mcal/h
+
+Linear Var Cost|Electricity|Coal:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Linear Var Cost|Electricity|Gas:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Linear Var Cost|Electricity|Nuclear:
+   description: Linear term (slope) of the heat rate straight line. Cost of the power generation unit who belongs to the specified power
+                plant category. It refers to the production cost of each energy unit (GWh, MWh,etc.).
+   unit: Mcal/GWh
+
+Linear Var Cost|Electricity|Oil:
```

Comment:
I am not sure to understand the meaning of "Linear term (slope) of the heat rate straight line". moreover from the rest of the definition seems this is the operation linear cost, no? Usually costs that are taken into account are either "investment costs" (when there is some change in the generation mix) or operation costs; both can be modelled as different kinds of functions (depending on the model), generally a simple quadratic or linear function is used.